### PR TITLE
[ci-visibility] Instructions for `datadog-ci` standalone binary

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do?
Include instructions for the usage of `datadog-ci`'s standalone binary.

### Motivation
Make it simpler to use `datadog-ci`, without needing to install `node` in your environment. 

### Preview
https://docs-staging.datadoghq.com/juan-fernandez/standalone-binary-datadog-ci/continuous_integration/setup_tests/junit_upload/#standalone-binary-beta

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/setup_tests/junit_upload.md [hunk 1]
```diff
@@ -30,6 +30,20 @@ Install the [`datadog-ci`][2] CLI globally using `npm`:
 npm install -g @datadog/datadog-ci
 {{< /code-block >}}
 
+### Standalone binary (beta)
+
+If installing nodejs in the CI is an issue, a standalone binary is provided with releases. Only _linux-x64_, _darwin-x64_ (macOS) and _win-x64_ (Windows) are supported at the time. To install for linux:
```
Comment (soyguijarro) [inline content/en/continuous_integration/setup_tests/junit_upload.md:35]: I would actually apply the architecture nomenclature you're using here to the download filenames:

- datadog-ci-linux -> datadog-ci_linux-x64
- datadog-ci-win -> datadog-ci_win-x64
- datadog-ci-macos -> datadog-ci_darwin-x64

File: content/en/continuous_integration/setup_tests/junit_upload.md [hunk 2]
```diff
@@ -30,6 +30,54 @@ Install the [`datadog-ci`][2] CLI globally using `npm`:
 npm install -g @datadog/datadog-ci
 {{< /code-block >}}
 
+### Standalone binary (beta)
+
+If installing nodejs in the CI is an issue, standalone binaries are provided with [releases][3]. Only _linux-x64_, _darwin-x64_ (MacOS) and _win-x64_ (Windows) are supported at the time. To install:
```
Comment (jtappa) [inline content/en/continuous_integration/setup_tests/junit_upload.md:35]:
```suggestion
If installing NodeJS in the CI is an issue, standalone binaries are provided with [Datadog CI releases][3]. Only _linux-x64_, _darwin-x64_ (MacOS) and _win-x64_ (Windows) are supported. To install, run the following from your CLI:
```

File: content/en/continuous_integration/setup_tests/junit_upload.md [hunk 3]
```diff
@@ -30,6 +30,54 @@ Install the [`datadog-ci`][2] CLI globally using `npm`:
...
+Then run any command with `Start-Process -FilePath "datadog-ci.exe"`:
+{{< code-block lang="powershell" >}}
+Start-Process -FilePath "./datadog-ci.exe" -ArgumentList version
+{{< /code-block >}}
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+<div class="alert alert-warning"><strong>Note</strong>: the standalone binaries are in <strong>beta</strong> and their stability is not guaranteed.</div>
```
Comment (jtappa) [inline content/en/continuous_integration/setup_tests/junit_upload.md:78]:
```suggestion
<div class="alert alert-warning"><strong>Note</strong>: The standalone binaries are in <strong>beta</strong> and their stability is not guaranteed.</div>
```

Comment (jtappa) [inline content/en/continuous_integration/setup_tests/junit_upload.md:78]: Move this up to just after the header for this section, ~line 35

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by jtappa — APPROVED** [review]

**Pull request merged**